### PR TITLE
[Doc](remote storage)Modify remote storage doc

### PR DESCRIPTION
**Pull request by Yukang-Lian**
# Versions 

- [x] dev
- [x] 3.0
- [x] 2.1
- [ ] 2.0

# Languages

- [x] Chinese
- [x] English

### CODE REVIEW

**Comment by Yukang-Lian** [comment]
run buildall

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/table-design/tiered-storage/remote-storage.md [hunk 1]
```diff
@@ -79,13 +47,25 @@ PROPERTIES
...
 );
+```
+
+:::tip
+创建 S3 RESOURCE 的时候，会进行 S3 远端的链接校验，以保证 RESOURCE 创建的正确。
+:::
 
+之后创建STORAGE POLICY，关联上文创建的RESOURCE：
+
+```Plain
```
Comment (yagagagaga) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/table-design/tiered-storage/remote-storage.md:58]:
```suggestion
```sql
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/table-design/tiered-storage/remote-storage.md [hunk 2]
```diff
@@ -79,13 +47,25 @@ PROPERTIES
...
 CREATE STORAGE POLICY test_policy
 PROPERTIES(
     "storage_resource" = "remote_s3",
     "cooldown_ttl" = "1d"
 );
+```
 
+最后建表的时候指定STORAGE POLICY：
+
+```Plain
```
Comment (yagagagaga) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/table-design/tiered-storage/remote-storage.md:68]:
```suggestion
```sql
```

**Comment by Yukang-Lian** [comment]
run buildall

**Review by yagagagaga — APPROVED** [review]

**Pull request merged**